__tests__/get-endpoints.test.js

```diff
@@ -47,4 +48,40 @@ test("handles invalid endpoints", () => {
         });
 });
 
+describe("GET /api/articles/:id", () => {
+    it("responds with an article for a valid ID", () => {
+        return request(app)
+            .get('/api/articles/1')
+            .expect(200)
+            .expect("Content-Type", /json/)
+            .then((result) => {
+                expect(Object.keys(result.body)).toHaveLength(8);
+                expect(result.body).toHaveProperty("author");
```

Comment:
if you're only getting back one article, your test would be better looking at the values on each keys using .toMatchObject